docs/extensions/overview.md

```diff
@@ -26,8 +14,41 @@ INSTALL 'fts';
 
-https://github.com/duckdb/duckdb/search?q=loaded_extensions
+## Listing core extensions
+
+You can check the list of core and installed extensions with the following query:
+```sql
+D select * From duckdb_extensions();
+┌──────────────────┬────────┬───────────┬──────────────────────────────────────────────────────────────────────────┬──────────────────────────────────────────────────────────────────────┐
+│  extension_name  │ loaded │ installed │                               install_path                               │                             description                              │
+├──────────────────┼────────┼───────────┼──────────────────────────────────────────────────────────────────────────┼──────────────────────────────────────────────────────────────────────┤
+│ fts              │ false  │ false     │                                                                          │ Adds support for Full-Text Search Indexes                            │
+│ httpfs           │ false  │ false     │                                                                          │ Adds support for reading and writing files over a HTTP(S) connection │
+│ icu              │ true   │ true      │ (BUILT-IN)                                                               │ Adds support for time zones and collations using the ICU library     │
+│ json             │ false  │ true      │ /Users/myth/.duckdb/extensions/da9ee490d/osx_arm64/json.duckdb_extension │ Adds support for JSON operations                                     │
+│ parquet          │ true   │ true      │ (BUILT-IN)                                                               │ Adds support for reading and writing parquet files                   │
+│ postgres_scanner │ false  │ false     │                                                                          │ Adds support for reading from a Postgres database                    │
+│ sqlite_scanner   │ false  │ false     │                                                                          │ Adds support for reading SQLite database files                       │
+│ substrait        │ false  │ false     │                                                                          │ Adds support for the Substrait integration                           │
+│ tpcds            │ false  │ false     │                                                                          │ Adds TPC-DS data generation and query support                        │
+│ tpch             │ true   │ true      │ (BUILT-IN)                                                               │ Adds TPC-H data generation and query support                         │
+└──────────────────┴────────┴───────────┴──────────────────────────────────────────────────────────────────────────┴──────────────────────────────────────────────────────────────────────┘
+```
+
+## All available extensions
```

Comment:
Perhaps replace this section with a table auto-generated from the `duckdb_extensions` function, e.g. (in the shell):

```sql
.mode markdown
select extension_name, description from duckdb_extensions();
```


|  extension_name  |                             description                              |
|------------------|----------------------------------------------------------------------|
| fts              | Adds support for Full-Text Search Indexes                            |
| httpfs           | Adds support for reading and writing files over a HTTP(S) connection |
| icu              | Adds support for time zones and collations using the ICU library     |
| json             | Adds support for JSON operations                                     |
| parquet          | Adds support for reading and writing parquet files                   |
| postgres_scanner | Adds support for reading from a Postgres database                    |
| sqlite_scanner   | Adds support for reading SQLite database files                       |
| substrait        | Adds support for the Substrait integration                           |
| tpcds            | Adds TPC-DS data generation and query support                        |
| tpch             | Adds TPC-H data generation and query support                         |

Reply:
Sure, only issue with that is the table-of-contents are seems to be auto appended by something